powershell/package.json

```diff
@@ -24,10 +24,13 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-task-lib#readme",
   "devDependencies": {
     "adm-zip": "^0.5.9",
+    "deasync": "^0.1.28",
     "mocha": "5.2.0",
     "q": "1.4.1",
     "shelljs": "^0.8.5",
-    "sync-request": "3.0.1",
     "typescript": "1.8.7"
+  },
+  "dependencies": {
+    "nodejs-file-downloader": "^4.11.1"
```

Comment:
probably it's better to keep it in devDependencies

Reply:
Replaced